crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -353,9 +365,36 @@ module sui::test_scenario {
         sui::transfer::share_object_impl(t)
     }
 
+    public fun get_objects_owned_by<T: key>(object_id: ID): vector<ID> {
```

Comment:
Is this just for receiving? I feel like this will be very easily conflated with dynamic fields